src/components/IconPicker/IconColorPicker.jsx

```diff
@@ -0,0 +1,80 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+
+import ClickAwayListener from 'cozy-ui/transpiled/react/ClickAwayListener'
+import GridList from 'cozy-ui/transpiled/react/GridList'
+import GridListTile from 'cozy-ui/transpiled/react/GridListTile'
+import Icon from 'cozy-ui/transpiled/react/Icon'
+import IconButton from 'cozy-ui/transpiled/react/IconButton'
+import Paper from 'cozy-ui/transpiled/react/Paper'
+import Popper from 'cozy-ui/transpiled/react/Popper'
+
+import { getIcon } from './IconIndex'
+
+import styles from '@/styles/folder-customizer.styl'
+
+export const ICON_COLORS = [
```

Comment:
Those colors are not shared with the previous folder colors? if yes we can reuse the previous ones